src/App.tsx

```diff
@@ -104,36 +127,27 @@ const App: FC = () => {
   };
 
   return (
-    <>
+    <OuterContainer>
+      <Header></Header>
       {errorPresent ? (
         <ErrorTextField>
           <b>{errorMessage}</b>
         </ErrorTextField>
       ) : (
         <Metadata marcData={marcData} />
       )}
-      {marcData && (
-        <Button
-          variant="contained"
-          color={showXMLPressed ? 'primary' : 'default'}
-          disableElevation={!showXMLPressed}
-          onClick={showXML}>
-          XML
-        </Button>
-      )}
-      {'  '}
-      {marcData && (
-        <Button
-          variant="contained"
-          color={showXMLPressed ? 'default' : 'primary'}
-          disableElevation={showXMLPressed}
-          onClick={showLineFormat}>
-          LineFormat
-        </Button>
+      {!errorPresent && marcData && (
+        <RadioContainer>
+          Velg format:
+          <input type="radio" value="xml" checked={showXMLPressed} onChange={showXML} />
+          <RadioLable onClick={showXML}>XML</RadioLable>
+          <input type="radio" value="linjeFormat" checked={!showXMLPressed} onChange={showLineFormat} />
+          <RadioLable onClick={showLineFormat}>Linjeformat</RadioLable>
```

Comment:
```suggestion
          <input id="xml" type="radio" value="xml" checked={showXMLPressed} onChange={showXML} />
          <RadioLable for="xml" onClick={showXML}>XML</RadioLable>
          <input id="linjeFormat" type="radio" value="linjeFormat" checked={!showXMLPressed} onChange={showLineFormat} />
          <RadioLable for="linjeFormat" onClick={showLineFormat}>Linjeformat</RadioLable>
```
Bør assosiere labels med inputen dens for screen-readers:
https://www.w3.org/WAI/tutorials/forms/labels/

Reply:
Kan jeg bruke aria-label? Ser ut til at styled-components.label ikke er særlig fan av "for='xml'". Liten erfaring med tilpassing til skjermlesere.

Reply:
Aria-label kan så absolutt brukes, slik jeg har forstått det er aria-labelby=id på label komponent å foretrekke over aria-label="forklaringa som allerede står i label komponenten"

Men ja override styled-components er noe herk. jeg kan vise hvordan jeg får det til hvis du blir sittende fast senere, men pcb nedstemmer alltid overridene mine og ber meg bruke "any" istedet fordi det blir seende så utrolig stygt ut.

Reply:
Da sattser jeg på labelledby. Er det noen bedre måte å teste dette på en skjermleseren på windows?

Reply:
Dette er også et argument du kan bruke til Asbjørn når han sier han ikke vil ha pen styling:
Material-ui ornder masse accessibility
 og masse annet krydder automatisk for deg:
https://material-ui.com/components/radio-buttons/

Reply:
Når det gjelder testing, jeg tester det ikke ved å kjøre koden gjennom noe program (f.eks skjermleser). Jeg tar heller å ser over at koden stemmer overens med kravene. Sjekk lista mi er som følger:
Overskrift hiearki: h1-h6. 
Brukerinput skal være forklart godt: både for selve teksten for "tilbakestående" og med aria-labels for de med dårlig syn.
Feedback tilbake til brukeren: progressbar og feilmeldinger skal det være tydelig hva sikter til. Hvis det er en spesifikk del av sida som har "feilet" eller ei progress" så slenges jeg på en aria-label. Igjen komponentene i Material-ui sier når du trenger å putte aria-label på.